### PR TITLE
improve code quality

### PR DESCRIPTION
**Pull request by Rohit25negi**

### CODE REVIEW

File: lamlight/source/service_router.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 import unzip_requirements
 
 
-import resources.constants as res_const
+import constants as res_const
```
Comment (sahil21) [inline lamlight/source/service_router.py]: @Rohit25negi Why this change? Relative imports are not recommended right?

File: lamlight/source/resources/example_resource.py [hunk 2]
```diff
@@ -3,6 +3,11 @@
 
 class ExampleResource(Resource):
```
Comment (harshitcodes) [inline lamlight/source/resources/example_resource.py]: Add documentation here for the class.
